### PR TITLE
add tensorflow 1.2 batchnorm mapping

### PR DESCRIPTION
**Pull request by yangw1234**
## What changes were proposed in this pull request?

add tensorflow 1.2 batchnorm mapping

## How was this patch tested?

add unit tests

### CODE REVIEW

**Comment by yiheng** [comment]
LGTM

**Comment by yangw1234** [comment]
pr validation pass
